src/app/modules/feedback/feedback.controller.ts

```diff
@@ -0,0 +1,143 @@
+import { celebrate, Joi, Segments } from 'celebrate'
+import { StatusCodes } from 'http-status-codes'
+import { ErrorDto, PrivateFormErrorDto } from 'shared/types'
+
+import { createLoggerWithLabel } from '../../config/logger'
+import { createReqMeta } from '../../utils/request'
+import { ControllerHandler } from '../core/core.types'
+import { PrivateFormError } from '../form/form.errors'
+import * as FormService from '../form/form.service'
+import * as PublicFormService from '../form/public-form/public-form.service'
+import { mapRouteError } from '../form/public-form/public-form.utils'
+import * as SubmissionService from '../submission/submission.service'
+
+import * as FeedbackService from './feedback.service'
+
+const logger = createLoggerWithLabel(module)
+
+const validateSubmitFormFeedbackParams = celebrate({
+  [Segments.BODY]: Joi.object()
+    .keys({
+      rating: Joi.number().min(1).max(5).cast('string').required(),
+      comment: Joi.string().allow('').required(),
+    })
+    // Allow other keys for backwards compability as frontend might put extra keys in the body.
+    .unknown(true),
+})
+
+export const submitFormFeedback: ControllerHandler<
```

Comment:
can we add jsdocs for this and also differentiate its name from the existing controller, even it's just something like `submitFormFeedbackV2`? will be super confusing to have two functions with the same name in different modules. we can also mark the old one as `@deprecated`

Reply:
Right okay, I named it the same because I thought we would remove the old one soon enough, but now I think your suggestion is better, thanks!